lib/LTP/WhiteList.pm

```diff
@@ -124,12 +131,22 @@ sub is_test_disabled {
 
 sub find_whitelist_testsuite {
     my ($suite) = @_;
+    my $issues;
 
-    my $path = get_var('LTP_KNOWN_ISSUES');
+    my $path = get_var('LTP_KNOWN_ISSUES_LOCAL');
     return undef unless defined($path) and -e $path;
 
     my $content = path($path)->slurp;
-    my $issues = Mojo::JSON::decode_json($content);
+
+    if (substr($content, 0, 1) eq '{') {
+        $issues = Mojo::JSON::decode_json($content);
+    } elsif (substr($content, 0, 3) eq '---') {
+        my $yp = YAML::PP->new;
+        $issues = $yp->load_string(decode_utf8 $content);
+    } else {
+        die "Only valid JSON and YAML are supported";
```

Comment:
This error message will stil look completely incomprehensible to anyone who runs into it. This is better:

    die "LTP_KNOWN_ISSUES must link to valid JSON or YAML file";